primitives/npos-elections/compact/src/lib.rs

```diff
@@ -66,10 +71,11 @@ pub(crate) fn syn_err(message: &'static str) -> syn::Error {
 /// for numbers will be used, similar to how `parity-scale-codec`'s `Compact` works.
 ///
 /// ```ignore
```

Comment:
sidenote: why is this test ignored ?

Reply:
not sure, can we bring it back @coriolinus?

Reply:
Sure: a2d3ac381

Reply:
sweet